src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/dashboard.scss

```diff
@@ -1,10 +1,13 @@
+// Images
+$dashboardLogoBackgroundImage: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB4bWxuczpza2V0Y2g9Imh0dHA6Ly93d3cuYm9oZW1pYW5jb2RpbmcuY29tL3NrZXRjaC9ucyIgd2lkdGg9IjEwMHB4IiBoZWlnaHQ9IjgwcHgiIHZpZXdCb3g9IjAgMCAxMDAgODAiIHZlcnNpb249IjEuMSI+PCEtLSBHZW5lcmF0b3I6IFNrZXRjaCAzLjMuMiAoMTIwNDMpIC0gaHR0cDovL3d3dy5ib2hlbWlhbmNvZGluZy5jb20vc2tldGNoIC0tPjx0aXRsZT5TaGFwZTwvdGl0bGU+PGRlc2M+Q3JlYXRlZCB3aXRoIFNrZXRjaC48L2Rlc2M+PGRlZnMvPjxnIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIHNrZXRjaDp0eXBlPSJNU1BhZ2UiPjxnIHNrZXRjaDp0eXBlPSJNU0FydGJvYXJkR3JvdXAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDAwMDAwLCAtOC4wMDAwMDApIiBmaWxsPSIjQ0NDQ0NDIj48ZyBza2V0Y2g6dHlwZT0iTVNMYXllckdyb3VwIj48cGF0aCBkPSJNOTkuNSA1My42IEMxMDAuNCA1Mi4yIDEwMCA1MC4xIDk4LjYgNDkuMSBMNzUgMzEuOSBDNzMuNiAzMC45IDcyLjMgMjguNiA3Mi4xIDI2LjkgTDcwLjQgMTEuMSBDNzAuMiA5LjQgNjguNyA4LjQgNjcuMSA5IEwyLjEgMzAuNyBDMC41IDMxLjMgLTAuNCAzMy4xIDAuMiAzNC43IEwxMi41IDcwLjkgQzEzIDcyLjYgMTQuOSA3NC4yIDE2LjYgNzQuNiBMNzUuNCA4Ny4zIEM3Ny4xIDg3LjcgNzkuMiA4Ni44IDgwLjEgODUuMyBMOTkuNSA1My42IEw5OS41IDUzLjYgWk00NiAzNy42IEM0NSAzOSA0Mi44IDM5LjggNDEuMSAzOS40IEwxMy40IDMzLjQgQzExLjcgMzMgMTEuNyAzMi4yIDEzLjQgMzEuNyBMNTkuMiAxNi4zIEM2MC45IDE1LjggNjEuNCAxNi41IDYwLjQgMTcuOSBMNDYgMzcuNiBMNDYgMzcuNiBaTTY0LjcgMTkuNiBDNjUuNyAxOC4xIDY2LjcgMTguNCA2Ni45IDIwLjEgTDcyLjUgNzQgQzcyLjcgNzUuNyA3MiA3NiA3MSA3NC41IEw0OS43IDQ1LjIgQzQ4LjcgNDMuOCA0OC43IDQxLjUgNDkuNyA0MC4xIEw2NC43IDE5LjYgTDY0LjcgMTkuNiBaTTYuNCAzOS4yIEM1LjggMzcuNSA2LjggMzYuNSA4LjUgMzYuOCBMNDAuNyA0My45IEM0Mi40IDQ0LjMgNDQuNiA0NS44IDQ1LjYgNDcuMiBMNjguOSA3OS4yIEM2OS45IDgwLjYgNjkuMyA4MS41IDY3LjYgODEuMSBMMjAgNzAuOCBDMTguMyA3MC40IDE2LjQgNjguNyAxNS45IDY3LjEgTDYuNCAzOS4yIEw2LjQgMzkuMiBaTTczLjQgMzkuMiBDNzMuMiAzNy40IDc0LjIgMzYuOSA3NS42IDM3LjkgTDkyLjYgNTAuMyBDOTQuMSA1MS4zIDk0LjUgNTMuMyA5My42IDU0LjggTDc5LjQgNzggQzc4LjUgNzkuNSA3Ny42IDc5LjMgNzcuNCA3Ny42IEw3My40IDM5LjIgTDczLjQgMzkuMiBaIiBza2V0Y2g6dHlwZT0iTVNTaGFwZUdyb3VwIi8+PC9nPjwvZz48L2c+PC9zdmc+) !default;
```

Comment:
You don't win anything so I would suggest moving the url to the declaration instead of the variable.